dcos/dcos-ui
# COPS-3968: fix Firefox table display issue

Removed min-width table-cell truncation hack because most browsers will obey the min-width: 0 style, causing the layout to break.

Tested in Firefox, Edge 17, Safari, and Chrome. Also tested to ensure text truncation still works

Closes COPS-3968

## Testing
**Test in Firefox or Edge**
1. Create a new service
2. Go to the "Environment" tab
3. Click "+ Add Environment Variable" and enter a key and a value
4. Click "Review & Run"

The table at the bottom of "Review & Run Service" should not have cells that are collapsed

## Trade-offs
None

## Dependencies
None

## Screenshots
Before:
<img width="933" alt="screen shot 2018-10-31 at 9 42 28 am" src="https://user-images.githubusercontent.com/2313998/47804387-e579e480-dcf1-11e8-9569-a47c127c1b81.png">


After:
<img width="921" alt="screen shot 2018-10-31 at 9 43 21 am" src="https://user-images.githubusercontent.com/2313998/47804393-e90d6b80-dcf1-11e8-98d4-802b05dc7ada.png">
